### PR TITLE
Update S3 config to get media and static files working

### LINKED ISSUE #199: Use new AWS account in production for custom and default profile photos
Continues work from big server change-over on Render to get profile photos working in production with Momentum AWS account.

When this work is done:

- [ ] a new user gets a new random photo using S3 images upon account creation
- [ ] a user’s new uploaded photo is stored in our S3 bucket
- [ ] mentor photos stored in S3 show up on the session signup page

### PR DESCRIPTION
**Pull request by amygori**
This moves us in the direction of using the Momentum S3 bucket for static and media files. Notable change here include:
- new `STORAGES` setting that replaces the old `DEFAULT_FILE_STORAGE` setting in Django versions prior to 4.2.
- new storage backend classes
- set the default ACL on AWS to `public-read`

**Caveats**
This needs to be tested against the front end before it's ready to deploy. No tests are included in this PR. 😬

### CODE REVIEW

File: config/settings.py [hunk 1]
```diff
@@ -156,23 +156,30 @@
...
+    AWS_SECRET_ACCESS_KEY = env('AWS_SECRET_ACCESS_KEY')
+    AWS_STORAGE_BUCKET_NAME = env('AWS_STORAGE_BUCKET_NAME')
     AWS_S3_CUSTOM_DOMAIN = f'{AWS_STORAGE_BUCKET_NAME}.s3.amazonaws.com'
     AWS_S3_OBJECT_PARAMETERS = {
         'CacheControl': 'max-age=86400',
     }
+    AWS_DEFAULT_ACL = 'public-read'
+    AWS_QUERYSTRING_AUTH = False
+    AWS_S3_FILE_OVERWRITE = False
+    STORAGES = {
```
Comment (GitLukeW) [inline config/settings.py:175]: The custom user model is setup to pull from django.core.files.storage import default_storage. With this new change I think we will need to update the import so it's pointing to storages now. 

To recreate the error you can run `python manage.py test`. This should fail any test looking for default storage.

**Review by GitLukeW — CHANGES_REQUESTED** [review]
Thank you for the contribution!!! Added one request to change an import name now that the storages name has changed.

File: config/settings.py (hunk 1 above)
Comment (amygori) [reply config/settings.py:175]: Ah nice! I will run the tests and check it out, and update the PR accordingly. Thanks for the heads up @GitLukeW!

**Comment by amygori** [comment]
[Use new AWS account in production for custom and default profile photos](https://trello.com/c/wdnTY5bJ/158-use-new-aws-account-in-production-for-custom-and-default-profile-photos)

**Comment by amygori** [comment]
Updated this PR to include tests 🎉! Changed settings in AWS aren't reflected in this PR but I think there was just a missing Read permission on existing objects in the bucket. We may want to update the way permissions are set, however, to be in line with [current AWS guidance](https://docs.aws.amazon.com/AmazonS3/latest/userguide/security-best-practices.html) to use a policy rather than ACLs. 

I manually tested against a local front-end instance and encountered a few bugs on the FE but  the requests appear to be working with the BE updated.

**Review by GitLukeW — APPROVED** [review]
Looks good, can't wait to get it into staging! Thank you for the contributing to the project!

**Pull request merged**